src/colorizer/ColorRamp.ts

```diff
@@ -1,15 +1,27 @@
-import { DataTexture, Color, ColorRepresentation, RGBAFormat, FloatType, LinearFilter } from "three";
+import { DataTexture, Color, ColorRepresentation, RGBAFormat, FloatType, LinearFilter, NearestFilter } from "three";
+
+export enum ColorRampType {
+  LINEAR,
+  HARD_STOP,
+}
```

Comment:
I struggled constantly between these being enums or booleans 😂
I don't *think* we'll have more gradients types in the future. I did like the enum here because it's easier to read imo.